ui/src/components/credentials/CredentialLinkQR.tsx

```diff
@@ -117,7 +117,7 @@ export function CredentialLinkQR() {
     : undefined;
 
   if (hasFailed) {
-    if (hasFailed.status === 404) {
+    if (hasFailed.type === "request-error" && hasFailed.error.status === 404) {
```

Comment:
This is not working, should be
```suggestion
   if (hasFailed.type === "request-error" && hasFailed.error.response.status === 404) {
```
![image](https://user-images.githubusercontent.com/19190248/236152053-14fc73df-df68-4381-90d7-520ff7f4fd3b.png)

Reply:
mmm... weird the changes in this PR should not make this fail.

Reply:
I see, you're right! [the previous implementation](https://github.com/0xPolygonID/sh-id-platform/pull/375/files#diff-04c057fe2e4d7a5a09df0be725644b5a8d5292db8be4d119016a0fae4f3f5067L66) was passing the content of `error.response`, not the whole error.

<img width="1038" alt="image" src="https://user-images.githubusercontent.com/15896177/236290665-3e7ebc83-5909-4cb7-ba2e-3b7ec08ac074.png">

Nice catch @adrifdez! Thx! 😅